document_page_tags/models/__init__.py

```diff
@@ -1,5 +1,3 @@
 # -*- coding: utf-8 -*-
-# Copyright 2015-2018 Therp BV <https://therp.nl>
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
But you can keep the license, although I would not insist on that. Only that otherwise you will get very bare init files, which are not pleasing to my eyes, but other people might think differently,